libs/client-api/src/http_publish.rs

```diff
@@ -192,6 +249,31 @@ impl Client {
       .into_data()
   }
 
+  #[instrument(level = "debug", skip_all)]
+  pub async fn get_default_published_collab<T>(
+    &self,
+    publish_namespace: &str,
+  ) -> Result<PublishInfoMeta<T>, AppResponseError>
+  where
+    T: serde::de::DeserializeOwned + 'static,
+  {
+    let url = format!(
+      "{}/api/workspace/published/{}",
+      self.base_url, publish_namespace,
+    );
+
+    let resp = self
+      .cloud_client
+      .get(&url)
+      .send()
+      .await?
+      .error_for_status()?;
+
+    AppResponse::<PublishInfoMeta<T>>::from_response(resp)
```

Comment:
Each api endpoint should call `log_request_id`.

Reply:
will add it in